scripts/aggregate-themes/theme-aggregator.js

```diff
@@ -53,7 +53,7 @@ class ThemeAggregator {
     const assets = ThemeAggregator.find(`**/themes/${themeName}/${file}`, options);
 
     // Add the dependency import if it exists.
-    assets.unshift(...ThemeAggregator.find(`${NODE_MODULES}${theme}/**/${file}`, options));
+    assets.unshift(...ThemeAggregator.find(`${NODE_MODULES}${themeName}/**/${file}`, options));
```

Comment:
should there be a test that verifies this fix behavior?

Reply:
Is this what caused the lib & src from being aggregated as well?

Reply:
Test case - https://github.com/cerner/terra-toolkit/pull/335/commits/8f3aa29281f762e4f25cb7c178fa3c1b69f1bddd No, this fixes a separate issue.

`theme` can either be the default theme or null. Without this change, we will either attempt to aggregate default node_module theme assets for every theme specified in config, or attempt to aggregate any theme file :(